crates/turborepo-lib/src/config/client.rs

```diff
@@ -133,72 +132,91 @@ mod test {
     #[test]
     fn test_client_arg_env_variable() -> Result<()> {
         #[derive(Debug)]
-        struct TestCase {
+        struct TestCase<'a> {
             arg: Option<u64>,
             env: String,
             output: u64,
+            want_err: Option<&'a str>,
```

Comment:
I think we'll only ever want to check against static error messages so this restriction would be fine.
```suggestion
        struct TestCase {
            arg: Option<u64>,
            env: String,
            output: u64,
            want_err: Option<&'static str>,
```